WalletWasabi.Gui/Controls/WalletExplorer/WalletExplorerViewModel.cs

```diff
@@ -71,6 +74,27 @@ public WalletExplorerViewModel()
 					}
 				});
 
+			CollapseAllCommand = ReactiveCommand.Create(() =>
+			{
+				foreach (var wallet in Wallets)
+				{
+					wallet.IsExpanded = false;
+				}
+			});
+
+			LurkingWifeModeCommand = ReactiveCommand.CreateFromTask(async () =>
```

Comment:
`Controls\WalletExplorer\WalletExplorerViewModel.cs(85,69): warning CS1998: This async method lacks 'await' operators and will run synchronously. Consider using the 'await' operator to await non-blocking API calls, or 'await Task.Run(...)' to do CPU-bound work on a background thread.`

Reply:
fixed